docs/ecosystem/ifttt/readme.md

```diff
@@ -11,71 +11,48 @@ meta:
 
 # IFTTT Integration for openHAB
 
-<div class="img-wrapper"><img src='./images/ifttt-meets-openhab.jpg' alt='IFTTT meets openHAB'/></div>
+[If This Then That (IFTTT)](https://ifttt.com) is a very popular web-based service that allows users to create "applets", which can combine different web services into powerful automation rules. With its [hundreds of services](https://ifttt.com/services), there are limitless new creative options: Be notified through text-to-speech if you are receive a DM on twitter, be warned through your lights if your favorite stock price falls below a certain threshold or put your house in away-mode when you turn on the ignition of your car - you can get further inspiration by browsing through existing applets.
```

Comment:
Small tidbit. Could you please have each sentence on its own line? The proper formatting is taken care of by the website builder.

Reply:
Hi Hakan, thanks for this, I changed it. No problem. Can you explain what difference between the two is?

Reply:
Hi Hakan, no problem, I changed it. Would you mind explaining what the difference is?

Reply:
If you change something small in one sentence, only that sentence is being shown as a diff in github / "git diff" / whatever too you use.

Basically, makes reviewing easier 😄